webpack/plugins.js

```diff
@@ -27,6 +28,7 @@ const BASE_PLUGINS = [
     }),
     new webpack.EnvironmentPlugin({
         GH_BUILD: !!process.env.GH_BUILD,
+        GOOGLE_API_KEY: process.env.GOOGLE_API_KEY || "AIzaSyAZ3ow-AhfTcOsBml7e3oXZ7JwqIATcGwU"
```

Comment:
i'm going to restrict this key to only simularium.allencell.org, so for local development, people can use their own api key if they want to use this feature.